`Pulumi.Tls` depends on alpha version of `Pulumi` dotnet nuget package

## Hello!
<!-- Please leave this section as-is, it's designed to help others in the community know how to interact with our GitHub issues. -->

- Vote on this issue by adding a 👍 reaction
- To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already)

## Issue details
[`Pulumi.Tls`](https://www.nuget.org/packages/Pulumi.Tls/4.2.0) version `4.2.0` depends on _alpha_ version (`3.29.0-alpha.1649268066`) of [`Pulumi`](https://www.nuget.org/packages/Pulumi) dotnet nuget package. This [breaks](https://github.com/gitfool/Pulumi.Dungeon/runs/5862745644?check_suite_focus=true#step:8:278) builds until the next stable `Pulumi` package is published:
```
  /__w/Pulumi.Dungeon/Pulumi.Dungeon/Aws/Aws.csproj : error NU1605: Detected package downgrade: Pulumi from 3.29.0-alpha.1649268066 to 3.28.0. Reference the package directly from the project to select a different version.  [/__w/Pulumi.Dungeon/Pulumi.Dungeon/Pulumi.Dungeon.sln]
  /__w/Pulumi.Dungeon/Pulumi.Dungeon/Aws/Aws.csproj : error NU1605:  Aws -> Pulumi.Tls 4.2.0 -> Pulumi (>= 3.29.0-alpha.1649268066)  [/__w/Pulumi.Dungeon/Pulumi.Dungeon/Pulumi.Dungeon.sln]
  /__w/Pulumi.Dungeon/Pulumi.Dungeon/Aws/Aws.csproj : error NU1605:  Aws -> Pulumi (>= 3.28.0) [/__w/Pulumi.Dungeon/Pulumi.Dungeon/Pulumi.Dungeon.sln]
  /__w/Pulumi.Dungeon/Pulumi.Dungeon/Cli/Cli.csproj : error NU1605: Detected package downgrade: Pulumi from 3.29.0-alpha.1649268066 to 3.28.0. Reference the package directly from the project to select a different version.  [/__w/Pulumi.Dungeon/Pulumi.Dungeon/Pulumi.Dungeon.sln]
  /__w/Pulumi.Dungeon/Pulumi.Dungeon/Cli/Cli.csproj : error NU1605:  pulumi-dungeon -> Aws -> Pulumi.Tls 4.2.0 -> Pulumi (>= 3.29.0-alpha.1649268066)  [/__w/Pulumi.Dungeon/Pulumi.Dungeon/Pulumi.Dungeon.sln]
  /__w/Pulumi.Dungeon/Pulumi.Dungeon/Cli/Cli.csproj : error NU1605:  pulumi-dungeon -> Aws -> Pulumi (>= 3.28.0) [/__w/Pulumi.Dungeon/Pulumi.Dungeon/Pulumi.Dungeon.sln]
      0 Warning(s)
      2 Error(s)
```

### Steps to reproduce
<!-- Provide a link to a live example or an unambiguous set of steps to reproduce this bug. Tell us what you expected to happen, and what happened instead. Please include code to reproduce, if relevant. -->

1. Upgrade to latest `Pulumi.Tls` package
2. Build breaks

Expected: 🚀 
Actual: 💥

--- a/provider/cmd/pulumi-resource-tls/schema.json
+++ b/provider/cmd/pulumi-resource-tls/schema.json
@@ -1097,7 +1097,7 @@
                 "tls": "Tls"
             },
             "packageReferences": {
-                "Pulumi": "3.*-*"
+                "Pulumi": "3.*"
             }
         },
         "go": {
@@ -1108,7 +1108,7 @@
         "nodejs": {
             "compatibility": "tfbridge20",
             "dependencies": {
-                "@pulumi/pulumi": "^3.0.0-alpha.0"
+                "@pulumi/pulumi": "^3.0.0"
             },
             "devDependencies": {
                 "@types/node": "^10.0.0"
@@ -1123,7 +1123,7 @@
             "compatibility": "tfbridge20",
             "readme": "\u003e This provider is a derived work of the [Terraform Provider](https://github.com/terraform-providers/terraform-provider-tls)\n\u003e distributed under [MPL 2.0](https://www.mozilla.org/en-US/MPL/2.0/). If you encounter a bug or missing feature,\n\u003e first check the [`pulumi/pulumi-tls` repo](https://github.com/pulumi/pulumi-tls/issues); however, if that doesn't turn up anything,\n\u003e please consult the source [`terraform-providers/terraform-provider-tls` repo](https://github.com/terraform-providers/terraform-provider-tls/issues).",
             "requires": {
-                "pulumi": "\u003e=3.0.0a1,\u003c4.0.0"
+                "pulumi": "\u003e=3.0.0,\u003c4.0.0"
             }
         }
     }

--- a/provider/resources.go
+++ b/provider/resources.go
@@ -80,7 +80,7 @@ func Provider() tfbridge.ProviderInfo {
 		},
 		JavaScript: &tfbridge.JavaScriptInfo{
 			Dependencies: map[string]string{
-				"@pulumi/pulumi": "^3.0.0-alpha.0",
+				"@pulumi/pulumi": "^3.0.0",
 			},
 			DevDependencies: map[string]string{
 				"@types/node": "^10.0.0", // so we can access strongly typed node definitions.
@@ -88,7 +88,7 @@ func Provider() tfbridge.ProviderInfo {
 		},
 		Python: &tfbridge.PythonInfo{
 			Requires: map[string]string{
-				"pulumi": ">=3.0.0a1,<4.0.0",
+				"pulumi": ">=3.0.0,<4.0.0",
 			},
 		},
 		Golang: &tfbridge.GolangInfo{
@@ -102,7 +102,7 @@ func Provider() tfbridge.ProviderInfo {
 		},
 		CSharp: &tfbridge.CSharpInfo{
 			PackageReferences: map[string]string{
-				"Pulumi": "3.*-*",
+				"Pulumi": "3.*",
 			},
 			Namespaces: map[string]string{
 				"tls": "Tls",

--- a/sdk/dotnet/Pulumi.Tls.csproj
+++ b/sdk/dotnet/Pulumi.Tls.csproj
@@ -45,7 +45,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Pulumi" Version="3.*-*" />
+    <PackageReference Include="Pulumi" Version="3.*" />
   </ItemGroup>
 
   <ItemGroup>

--- a/sdk/nodejs/package.json
+++ b/sdk/nodejs/package.json
@@ -14,7 +14,7 @@
         "install": "node scripts/install-pulumi-plugin.js resource tls ${VERSION}"
     },
     "dependencies": {
-        "@pulumi/pulumi": "^3.0.0-alpha.0"
+        "@pulumi/pulumi": "^3.0.0"
     },
     "devDependencies": {
         "@types/node": "^10.0.0",

--- a/sdk/python/setup.py
+++ b/sdk/python/setup.py
@@ -60,7 +60,7 @@ setup(name='pulumi_tls',
       },
       install_requires=[
           'parver>=0.2.1',
-          'pulumi>=3.0.0a1,<4.0.0',
+          'pulumi>=3.0.0,<4.0.0',
           'semver>=2.8.1'
       ],
       zip_safe=False)

